applications/GeoMechanicsApplication/geo_mechanics_application.cpp

```diff
@@ -256,8 +264,6 @@ void KratosGeoMechanicsApplication::Register() {
     KRATOS_REGISTER_CONDITION("LineNormalLoadDiffOrderCondition2D4N", mLineNormalLoadDiffOrderCondition2D4N)
     KRATOS_REGISTER_CONDITION("LineNormalLoadDiffOrderCondition2D5N", mLineNormalLoadDiffOrderCondition2D5N)
     KRATOS_REGISTER_CONDITION("LineNormalFluidFluxDiffOrderCondition2D3N", mLineNormalFluidFluxDiffOrderCondition2D3N)
-    KRATOS_REGISTER_CONDITION("LineNormalFluidFluxDiffOrderCondition2D4N", mLineNormalFluidFluxDiffOrderCondition2D4N)
-    KRATOS_REGISTER_CONDITION("LineNormalFluidFluxDiffOrderCondition2D5N", mLineNormalFluidFluxDiffOrderCondition2D5N)
```

Comment:
Should these 2 be removed? Do they also not work correctly for these already existing diff order conditions?

Reply:
@mnabideltares Could you please add the answer to this question here, such that it's documented?

Reply:
I will handle these two conditions (and two other from Axisymmetric which are removed in revision), in a coming issue related to diff_order conditions. 

Because the locations of the nodes do not match for 15 and 10, as well as, for 10 and 6 elements, special treatment need to be taken into account. For the conditions, it is also same. The diff-order conditions do not match for 5 and 4 line conditions. I think the element and condition need to be fixed together so that they use the same ordering and same interpolation schemes.

Here, if the conditions are registed in Kratos, it can happern that a user unaware of this issue, tries to use. The danger of using it is that it works but it gives wrong results. So, I decided to remove them temporarily from the list to not give opportunity to be used at the current state. Next sprint, these conditions, together with diff_order elements will be handeled and conditions will be added.